source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
+}
+
+.figure__image {
```

Comment:
Think the `<picture>` needs styling too as it seems to have some block-end margin.

Reply:
It's weirder than that: `<picture>` is sitting below its `<img>` child:

<img width="685" alt="Screenshot 2019-04-30 10 21 51" src="https://user-images.githubusercontent.com/2893480/56952363-19fc1b00-6b32-11e9-9f32-50bffb8f5c82.png">